spec/factories/symbol_factory.rb

```diff
@@ -21,13 +21,27 @@
     kind { FactoryGirl.generate :symbol_kind }
     name { FactoryGirl.generate :name }
 
-    after(:build) do |symbol|
-      symbol.locid = "#{symbol.ontology.locid}//#{symbol.name}"
+    after(:create) do |symbol|
+      LocId.where(
+                    locid: "#{symbol.ontology.locid}//#{symbol.name}",
+                  ).first_or_create!(
+                  assorted_object_id: symbol.id,
+                  assorted_object_type: 'Sentence',
```

Comment:
This should not be `'Sentence'` because it's in the *symbol* factory.
I fixed this in 5a76ecf3e9542845fe05f65e722394fd5228a85a.